sign.js

```diff
@@ -219,6 +219,9 @@ var signAsync = module.exports.signAsync = function (opts) {
       var promise
       if (opts.identity) {
         debuglog('`identity` passed in arguments.')
+        if (opts.skipIdentityValidation) {
```

Comment:
@develar yea camel-case should work better with the programmable interface.

As a side question before publishing the module, would you prefer `identity-validation` or `validate-identity` as the option name?